docs/source-pytorch/extensions/strategy.rst

```diff
@@ -113,12 +113,6 @@ There are powerful third-party strategies that integrate well with Lightning but
    * - Name
      - Package
      - Description
-   * - ColossalAI
```

Comment:
This table now contains only one row. I think you can just delete it and do the same as above, linking to the new page.